packages/autocomplete/src/examples/autocomplete.md

```diff
@@ -0,0 +1,175 @@
+optionally allow users to add content with the "Add" Menu Item.
+
+```jsx
+const { MenuView, Item, AddItem, Separator } = require('@zendeskgarden/react-menus');
+const { TextGroup, Label, FauxInput, Input } = require('@zendeskgarden/react-textfields');
+const { FieldContainer } = require('@zendeskgarden/react-selection');
+
+const NoItemsMessage = styled.div`
+  margin: 16px;
+  text-align: center;
+`;
+
+const stringContains = (original, comparison) => {
+  const formattedOriginal = original.replace(/ /g, '').toLocaleLowerCase();
+  const formattedComparison = comparison.replace(/ /g, '').toLocaleLowerCase();
+
+  return formattedOriginal.indexOf(formattedComparison) !== -1;
+};
+
+const getMatchingMenuItems = (searchValue, selectedValue, getItemProps, focusedKey) => {
+  const menuItems = state.natoPhonetics
+    .filter(phonetic => stringContains(phonetic, searchValue))
+    .map(phonetic => (
+      <Item
+        {...getItemProps({
+          key: phonetic,
+          focused: focusedKey === phonetic,
+          checked: selectedValue === phonetic,
+          'aria-selected': selectedValue === phonetic
+        })}
+      >
+        {phonetic}
+      </Item>
+    ));
+
+  return menuItems.length === 0 ? <NoItemsMessage>No items found</NoItemsMessage> : menuItems;
+};
+
+initialState = {
+  value: 'Default value',
+  inputValue: '',
+  selectedKeys: {},
+  isFocused: false,
+  isOpen: false,
+  natoPhonetics: [
+    'Alfa',
+    'Bravo',
+    'Charlie',
+    'Delta',
+    'Echo',
+    'Foxtrot',
+    'Golf',
+    'Hotel',
+    'India',
+    'Juliett',
+    'Kilo',
+    'Lima',
+    'Mike',
+    'November',
+    'Oscar',
+    'Papa',
+    'Quebec',
+    'Romeo',
+    'Sierra',
+    'Tango',
+    'Uniform',
+    'Victor',
+    'Whiskey',
+    'X-ray',
+    'Yankee',
+    'Zulu'
+  ]
+};
+
+<FieldContainer>
+  {({ getLabelProps, getInputProps: getFieldInputProps }) => (
+    <TextGroup>
+      <Label {...getLabelProps()}>Basic Autocomplete</Label>
+      <AutocompleteContainer
+        isOpen={state.isOpen}
+        onSelect={selectedKey => {
+          let natoPhonetics = state.natoPhonetics;
+
+          if (selectedKey === 'add-item') {
+            natoPhonetics = natoPhonetics.slice();
+            natoPhonetics.push(state.inputValue);
+            selectedKey = state.inputValue;
+          }
+
+          setState({ value: selectedKey, natoPhonetics, inputValue: '' });
+        }}
+        onStateChange={newState => {
+          let inputValue = state.inputValue;
+
+          if (typeof newState.isOpen !== 'undefined' && !newState.isOpen) {
+            newState.inputValue = '';
+          }
+
+          setState(newState);
+        }}
+        trigger={({ getTriggerProps, getInputProps, triggerRef, inputRef, isOpen }) => {
+          return (
+            <FauxInput
+              {...getTriggerProps({
+                open: isOpen,
+                focused: state.isFocused || isOpen,
+                select: true,
```

Comment:
I'm not exactly sure how this will work in the new container world, but we agreed that a `<select>` should retain mouse control for collapsing an expanded menu with our previous https://garden.zendesk.com/react-components-archive/#!/MultiSelect. We should model that here.

Reply:
This is now fixed. Just missed this interaction when bringing over the new functionality.